AssessmentService/src/main/java/com/revature/caliber/intercoms/TraineeClient.java

```diff
@@ -9,7 +9,7 @@
 
 
 //TODO: No way to get trainee by id in UserService?
-@FeignClient(name="trainee", url="localhost:9090/")
+@FeignClient(name="trainee", url="localhost:9090/",fallback=TraineeClientFallback.class)
```

Comment:
For the Feign Clients we would like to specify the url in config files not in the code since the url will need to be changed when deployed.